dist/models/Players.js

```diff
@@ -3,23 +3,24 @@
 const mongoose = require("mongoose");
 const PlayerSchema = mongoose.Schema({
   username: {
-    type: Number,
+    type: String,
     required: true
   },
   level: {
-    type: Number,
+    type: String,
     required: false
   },
   score: {
-    type: Number,
+    type: String,
     required: false
   },
   email: {
-    type: Number,
-    required: true
+    type: String,
+    required: false,
+    default: 'email@email.com'
   },
   group_id: {
-    type: Number,
+    type: String,
     required: false
   }
 });
```

Comment:
this is the result of running `npm run build`

Reply:
The manual change happened in `src/models/Players.js`....

Then after running `npm run build` the `dist/` directory was updated

So to answer your question: it was a manual change in `src/` that initiated the change in `dist/` after running `npm run build`

Unless i'm misunderstanding the flow of ES6 -> ES5